### ISSUE TITLE
csrf_token in templates/connect_button.html

### ISSUE THREAD

**Issue by yeago** [ryanmark/django-facebookconnect#2]
in prep for 1.2, this needs to be there or else 403. Would be a handy default.

**Comment by philotas** [comment]
yep that is needed for 1.2 if protection is turned on. do you know if you have to put it somewhere else? in order for the connect to work? at the moment putting {% csrf_token %} did the trick for me.

**Comment by yeago** [comment]
yeah, I did the same.
